src/area-chart/model/index.ts

```diff
@@ -19,7 +19,7 @@ export interface ChartModel<T extends AreaChartProps.DataTypes> {
     onSVGMouseMove: (event: React.MouseEvent<SVGElement>) => void;
     onSVGMouseOut: (event: React.MouseEvent<SVGElement>) => void;
     onSVGMouseDown: (event: React.MouseEvent<SVGSVGElement>) => void;
-    onSVGKeyDown: (event: React.KeyboardEvent) => void;
```

Comment:
@pan-kot Updated this typing for compatibility with the handleKey utility. There is a unit test that binds the keydown to a div, but the code binds it to an svg.